rwgame/RWGame.cpp

```diff
@@ -797,4 +968,10 @@ void RWGame::globalKeyEvent(const SDL_Event& event)
 		break;
 	default: break;
 	}
+
+	std::string keyName = SDL_GetKeyName(event.key.keysym.sym);
```

Comment:
I'm aware of it and I initially planned to use it. However, I read many people were having problems with it.
https://wiki.libsdl.org/SDL_StartTextInput < Note that this also activates an onscreen keyboard.

I'm aware that my way of collecting input is pretty hacky though, but I feel it's an okay solution which is hopefully portable.
Cheats are not too important anyway, so even if this breaks it's not serious. We'll worry about it once people report issues.